docs/guide/serialization.rst

```diff
@@ -0,0 +1,94 @@
+#############
+Serialization
```

Comment:
Not sure about naming this Serialization. Maybe there's some better name for this page?

Reply:
This page covers how to serialize data to other formats (cairo, python). I think it is a good name. Will be even a better fit once new serialization module is merged.
Any suggestions?

Reply:
Let's keep it as `Serialized` for now. We can change it if there's some confusion about it.